src/ResourceManagement/Resource/Microsoft.Azure.Management.ResourceManager/Utility/SafeJsonConvertWrapper.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Microsoft.Azure.Management.ResourceManager
+{
+    using Microsoft.Azure.Management.ResourceManager.Models;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    public static class SafeJsonConvert
+    {
+        public static string SerializeObject(object obj, JsonSerializerSettings settings)
+        {
+            if (obj != null && obj is Deployment)
```

Comment:
No, DeploymentExtended is the return type. When we create deployment, we create object of type Deployment.